### ISSUE TITLE
minimega: report cc commands issued

### ISSUE THREAD

**Issue by jcrussell** [sandia-minimega/minimega#1330]
Now that we apply `cc filter` on the server, we can count how many times we issue the command. Expose this as a column to users so that they can compare it with number of responses received.

**Comment by djfritz** [comment]
@jcrussell was this closed by #1346?

**Comment by jcrussell** [comment]
Nope, this is to count the number of times that minimega sends a command to the VMs. It gets a little weird if the VM reboots and minimega resends the command.
